controls.py

```diff
@@ -5,64 +5,57 @@
 
 
 ######################################Creation Functions
-def create_student(username, name, password, email, token=""):
+def create_account(username):
     # Create a Student and Account instance in the rd database.
     s = db.get_session()
 
-    Acc =  Account(username=username, name=name, password=password, email=email, account_token=token)
+    Acc =  Account(username)
     s.add(Acc)
-    s.flush()
-
-    Stu = Student(related_account=Acc.id)
-    s.add(Stu)
     s.commit()
     s.close()
+
     print(username, "added to the database")
 
-def create_donor(username, password, email, token=""):
-    # Create a Donor and Account instance in the rd database
+def approve_admin(usernameAttempt):
     s = db.get_session()
-
-    Acc =  Account(username=username, password=password, email=email, account_token=token)
-    s.add(Acc)
-    s.flush()
-
-    Don = Donor(related_account=Acc.id)
-    s.add(Don)
+    A = s.query('accounts').filter(Account.username==usernameAttempt).first()
+    A.admin = True
     s.commit()
     s.close()
 
+def approve_requesting(usernameAttempt):
+    s = db.get_session()
+    A = s.query('accounts').filter(Account.username==usernameAttempt).first()
+    A.approved = True
+    s.commit()
+    s.close()
 
-def open_request(student_id, amount, reason, title="Donate to a Reedie in needie", anon=True, amount_filled=0.0):
+def open_request(student_id, amount, reason, title, anon):
     # Create a request instance in the rd database.
     s = db.get_session()
-    req = Request(requested_by=student_id, amount_needed=amount, title=title, amount_filled=amount_filled, description=reason, anon=anon)
+    if title == '': title = "Donate to a Reedie in needie"

```

Comment:
I agree that everyone should title theirs, but I feel like this title is silly enough to push people to not forget it!